datafusion/physical-plan/src/topk/mod.rs

```diff
@@ -197,6 +302,9 @@ impl TopK {
             row_converter: _,
             scratch_rows: _,
             mut heap,
+            common_prefix_converter: _,
+            common_sort_prefix: _,
+            finished: _,
```

Comment:
🤔 I wonder why the spread operator was just not used here on the first place. For consistency it might make more sense to leave it as is.

```suggestion
            ..
```